Sources/FingerprintJS/SystemControl/Subsystems/HardwareSubsystem.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+
+struct HardwareSubsystem {
+    private let systemControl: SystemControlValuesRetrieving
+
+    init(systemControl: SystemControlValuesRetrieving = SystemControl()) {
+        self.systemControl = systemControl
+    }
+}
+
+extension HardwareSubsystem {
+    private struct Definition {
```

Comment:
Since `Definition` is essentially being used as a namespace, we would be better of with making it a caseless enum, which cannot be instantiated.